src/Profile.php

```diff
@@ -4015,6 +4075,33 @@ public static function dropdownHelpdeskItemtypes($options)
         return Dropdown::showFromArray($p['name'], $values, $p);
     }
 
+    /**
+     * @return array<int, string>
+     **/
+    public static function getMentionsLists(bool $disabled): array
```

Comment:
Not a mandatory changes.

```suggestion
    public static function getUseMentionsChoices(bool $disabled): array
```

Reply:
This method is used only on the current class, please make it private.